src/main/jbake/content/docs/user/installation.adoc

```diff
@@ -24,12 +24,21 @@ Fortunately, this only requires setting
 a few system properties.
```

Comment:
> "requires setting a few environment variables" if you no longer want 
> to talk about sys props

Thx

It is not that I don't want, but that 
`-Dhawkular-metrics.cassandra-nodes` only
applies to metrics, but not to alerts or inventory.